room/room-compiler-processing/src/main/java/androidx/room/compiler/processing/ksp/KspBasicAnnotationProcessor.kt

```diff
@@ -36,18 +35,21 @@ abstract class KspBasicAnnotationProcessor(
     val symbolProcessorEnvironment: SymbolProcessorEnvironment
 ) : SymbolProcessor, XBasicAnnotationProcessor {
 
+    private val xEnv = KspProcessingEnv(
+        symbolProcessorEnvironment.options,
+        symbolProcessorEnvironment.codeGenerator,
+        symbolProcessorEnvironment.logger
+    )
+
+    final override val xProcessingEnv: XProcessingEnv = xEnv
```

Comment:
Similar to JavacBasicAnnotationProcessor, can be a getter property:
`final override val xProcessingEnv: XProcessingEnv get() = xEnv`

Reply:
Done.